### ISSUE TITLE
Expose multiple versions of the stack in the devfile registry viewer

### ISSUE THREAD

**Issue by elsony** [devfile/api#950]
/kind user-story

**Which area this user story is related to?**
/area registry

## User Story
As a developer, I want to view different versions of the stack in the devfile registry viewer, so that see all the stacks that are available in the public community registry.

## Acceptance Criteria
- [ ] Expose multiple versions of the stack in the devfile registry viewer on the page that displays the individual stack
- [ ] Allow user to switch between different versions of the stack
- [ ] The default version of the stack is selected by default as specified by the stack provider
- [ ] Provide a filter on the main page to view stacks/samples based on the schema specification (lower priority)

**Comment by johnmcollier** [comment]
https://github.com/devfile/devfile-web/pull/40

**Comment by johnmcollier** [comment]
Blocked by https://github.com/devfile/api/issues/747
